.travis.yml

```diff
@@ -1,24 +1,70 @@
----
 language: python
 
-python:
-    - "3.6"
+cache: pip
+env: TRAVIS=1
```

Comment:
I did like the different versions, but @northwestwitch is probably right. Let's keep it simple. There will be an option to change this when switching to git tasks at some point.

Reply:
Then maybe you would have liked [this](https://travis-ci.org/Clinical-Genomics/scout/builds/637346864)?